live/src/config.sh

```diff
@@ -61,11 +61,16 @@ systemctl enable checkmedia.service
 systemctl enable qemu-guest-agent.service
 systemctl enable setup-systemd-proxy-env.path
 systemctl enable x11-autologin.service
-systemctl enable spice-vdagentd.service
+test -f  /usr/lib/systemd/system/spice-vdagentd.service && systemctl enable spice-vdagentd.service
```

Comment:
I kind of expect that identical thing should happen to qemu-guest-agent..and also x11-autologin